docs/API/panel/trend/index.md

```diff
@@ -378,6 +380,28 @@ PARAMETERS:
 * **value** (`object`)
 
 TODO docs
```

Comment:
todo? We've now got docs?

Reply:
That comes from the upstream schemas in grafana/grafana, authors of that panel should document this.